tests/strictequal.test.js

```diff
@@ -0,0 +1,52 @@
+const describe = require('mocha').describe
```

Comment:
Since strictEquals is about the same type and the same value I would expect 4 tests

1) Same type, same value (ex. 10, 10)
2) Same type, different value (ex. 10, 15)
3) Different type, same value (ex. 10, '10')
4) Different type, different value (ex. 10, '15')

Reply:
I only performed three tests because i am testing for different types first. Sine there are only two possibilities for the type test the minimum number of test to evaluate the accuracy is three.

Reply:
Each of these four test types I laid out are necessary as otherwise you open up the possibility of edge cases creeping in. Different type, same value and different type, different value are distinct test cases and should both be covered.